include/circt/Dialect/FIRRTL/AnnotationDetails.h

```diff
@@ -97,6 +99,8 @@ constexpr const char *prefixInterfacesAnnoClass =
     "sifive.enterprise.grandcentral.PrefixInterfacesAnnotation";
 constexpr const char *augmentedGroundTypeClass =
     "sifive.enterprise.grandcentral.AugmentedGroundType"; // not an annotation
+constexpr const char *augmentedBundleTypeClass =
+    "sifive.enterprise.grandcentral.AugmentedBundleType"; // not an annotation
```

Comment:
This can be cleaned up following, but these could be defined as static methods of the ODS definitions of these instead of in `AnnotationDetails.h`. See: https://github.com/llvm/circt/blob/main/include/circt/Dialect/FIRRTL/FIRRTLAttributes.td#L91

For background, this came about because there is a lot of matching in GCT Views based on the type. This started as hand-rolled LLVM-style RTTI, but it was cleaner to put this into ODS and let it generate the RTTI boilerplate. This, along with the GCT Views member function `fromAttr` are early prototypes of what a more structured approach to Annotations could look like.